Taarafo.Core/Brokers/Storages/IStorageBroker.GroupMemberships.cs

```diff
@@ -11,5 +12,6 @@ namespace Taarafo.Core.Brokers.Storages
     public partial interface IStorageBroker
     {
         ValueTask<GroupMembership> InsertGroupMembershipAsync(GroupMembership groupMembership);
+        ValueTask<GroupMembership> SelectGroupMembershipByIdAsync(Guid groupMembership);
```

Comment:
change this to: `ValueTask<GroupMembership> SelectGroupMembershipByIdAsync(Guid id);`
your variables should be descriptive, groupMembership would be more fitting for the object,  we are just using the Id here.